package.json

```diff
@@ -4,25 +4,37 @@
     "description": "This repository is made by students of Code Chrysalis",
     "main": "index.js",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start": "node server.js",
+        "webpack": "webpack --watch",
+        "hack": "webpack-dev-server",
+        "postinstall": "webpack"
```

Comment:
I would still suggest a separate build step that runs webpack and have this one run `yarn build`